DOCS-2929: Update changelog (#3459)
viamrobotics/docs: docs/appendix/changelog.md

<!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->

{{% changelog date="2024-09-06" color="added" title="Sample datasets" %}}

You can now get a sample dataset from the docs.
For more information, see [Sample dataset](/services/data/dataset/#sample-dataset).

{{% /changelog %}}

{{% changelog date="2024-08-26" color="added" title="ESP32 cameras" %}}

`viam-micro-server` now supports cameras on ESP32s.
For more information, see [Configure an esp32-camera](/components/camera/esp32-camera/).

{{% /changelog %}}

{{% changelog date="2024-08-26" color="changed" title="Micro-RDK now called viam-micro-server" %}}

> The micro-RDK is now referred to as `viam-micro-server`.

Reviewer: ```suggestion
The lightweight version of `viam-server` that is built from the micro-RDK is now referred to as `viam-micro-server`.
```

Author: good suggestion!

For more information, see [viam-micro-server](/architecture/viam-micro-server/).

{{% /changelog %}}

{{% changelog date="2024-08-26" color="added" title="Provisioning" %}}

You can now configure provisioning for machines with the Viam Agent.
For more information, see [Configure provisioning with viam-agent](/how-tos/provision-setup/).

{{% /changelog %}}

{{% changelog date="2024-08-16" color="added" title="Data capture for vision" %}}

Data capture is now possible for the vision service.
For more information, see [Supported components and services](/services/data/capture/#supported-components-and-services).

{{% /changelog %}}

{{% changelog date="2024-08-01" color="added" title="Create custom training scripts" %}}

> You can now create, upload and submit custom training scripts to the Viam Registry.

Reviewer: That doesn't make sense, you submit jobs not trainign scripts
```suggestion
You can now upload custom training scripts to the Viam Registry and use them to train machine learning models.
```

Author: thanks

For more information, see [Create custom training scripts](/how-tos/create-custom-training-scripts/).

{{% /changelog %}}

{{% changelog date="2024-07-19" color="changed" title="Operators can now view data" %}}

The operator role now has view permissions for the data in the respective resource a user has access to.

> For more information, see [Data and machine learning](/cloud/rbac/#data-and-machine-learning).

Reviewer: That link text may be confusing without added specificity.
```suggestion
For more information, see [Data and machine learning permissions](/cloud/rbac/#data-and-machine-learning).
```
